ggml.c

```diff
@@ -2967,6 +3038,75 @@ static void ggml_vec_dot_q5_0_q8_0(const int n, float * restrict s, const void *
     }
 
     *s = hsum_float_8(acc);
+#elif defined(__riscv_v_intrinsic)
+    float sumf = 0.0;
+
+    uint32_t qh;
+
+    uint32_t temp_1[16] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15};
+    uint32_t temp_2[16] = {0x1,   0x2,   0x4,   0x8,   0x10,   0x20,   0x40,   0x80, 
+                         0x100, 0x200, 0x400, 0x800, 0x1000, 0x2000, 0x4000, 0x8000};
```

Comment:
Not exactly, these values are not in additional code and I am using it for AND operation to extract a single bit from qh. In ggml they have provided the table for 128 bit which I can't use for 256 bit vector, so i make these temporary variable instead of using that table multiple time for masking

Reply:
Cant you use bit shift and mask with 0x1?

Reply:
Its ok i saw the intrinsic requirement.